cfn-templates/data-exports-aggregation.yaml

```diff
@@ -451,8 +475,9 @@ Resources:
         RefreshCadence:
           Frequency: "SYNCHRONOUS"
 
+  # FOCUS
 
-  ## Deploy FOCUS natively via CFN resource if we are regions that support native CFN
+  ## Deploy Data Export natively via CFN resource if we are regions that support native CFN
```

Comment:
Still need to change wording to "## Deploy Data Export natively via CFN resource in regions that support native CFN"